ngi_pipeline/database/filesystem.py

```diff
@@ -160,9 +166,18 @@ def create_charon_entries_from_project(project, best_practice_analysis="whole_ge
                                      'seqrun "{}" already exists; next...'.format(project, sample,
                                                                                   libprep, seqrun))
                     else:
+                        update_failed=True
                         LOG.error(e)
                         continue
 
+    if update_failed :
+        if retry_on_fail:
+        create_charon_entries_from_project(project, best_practice_analysis=best_practice_analysis,
+                                       sequencing_facility=sequencing_facility
+                                       force_overwrite=force_overwrite, delete_existing=delete_existing,
+                                       retry_on_fail=False):
```

Comment:
this is tail recursion.... which is  a particular case of recursion.
but ok, it is correct and should work as intended. did you checked the indentation problem? if that is fine I can merge